jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/AssetFileGenerator.java

```diff
@@ -170,6 +172,52 @@ public class AssetFileGenerator {
 	RDSDBManager rdsdbManager;
 
+	private static final String TARGETTYPE_VIRTUALMACHINE = "virtualmachine";
+	private static final String TARGETTYPE_VIRTUALMACHINESCALESET = "virtualmachinescaleset";
+	private static final String TARGETTYPE_STORAGEACCOUNT = "storageaccount";
+	private static final String TARGETTYPE_SQLDATABASE = "sqldatabase";
+	private static final String TARGETTYPE_NSG = "nsg";
+	private static final String TARGETTYPE_DISK = "disk";
+	private static final String TARGETTYPE_NETWORKINTERFACE = "networkinterface";
+	private static final String TARGETTYPE_VNET = "vnet";
+	private static final String TARGETTYPE_LOADBALANCER = "loadbalancer";
+	private static final String TARGETTYPE_SQLSERVER = "sqlserver";
+	private static final String TARGETTYPE_SECURITYCENTER = "securitycenter";
+	private static final String TARGETTYPE_BLOBCONTAINER = "blobcontainer";
+	private static final String TARGETTYPE_RESOURCEGROUP = "resourcegroup";
+	private static final String TARGETTYPE_COSMOSDB = "cosmosdb";
+	private static final String TARGETTYPE_MYSQLSERVER = "mysqlserver";
+	private static final String TARGETTYPE_DATABRICKS = "databricks";
+	private static final String TARGETTYPE_MARIADB = "mariadb";
+	private static final String TARGETTYPE_POSTGRESQL = "postgresql";
+	private static final String TARGETTYPE_SNAPSHOT = "snapshot";
+	private static final String TARGETTYPE_PUBLICIPADDRESS = "publicipaddress";
+	private static final String TARGETTYPE_ROUTETABLE = "routetable";
+	private static final String TARGETTYPE_SECURITYALERTS = "securityalerts";
+	private static final String TARGETTYPE_POLICYEVALUATIONRESULTS = "policyevaluationresults";
+	private static final String TARGETTYPE_SITES = "sites";
+	private static final String TARGETTYPE_VAULTS = "vaults";
+	private static final String TARGETTYPE_WORKFLOWS = "workflows";
+	private static final String TARGETTYPE_BATCHACCOUNTS = "batchaccounts";
+	private static final String TARGETTYPE_NAMESPACES = "namespaces";
+	private static final String TARGETTYPE_SEARCHSERVICES = "searchservices";
+	private static final String TARGETTYPE_SUBNETS = "subnets";
+	private static final String TARGETTYPE_REDISCACHE = "rediscache";
+	private static final String TARGETTYPE_ACTIVITYLOG = "activitylog";
+	private static final String TARGETTYPE_SECURITYPRICINGS = "securitypricings";
+	private static final String TARGETTYPE_WEBAPP = "webapp";
+	private static final String TARGETTYPE_SUBSCRIPTION = "subscription";
+	private static final String TARGETTYPE_FUNCTIONAPP = "functionapp";
+	private static final String TARGETTYPE_MYSQLFLEXIBLE = "mysqlflexible";
+	private static final String TARGETTYPE_DIAGNOSTICSETTING = "diagnosticsetting";
+	private static final String TARGETTYPE_BLOBSERVICE = "blobservice";
+	private static final String TARGETTYPE_KUBERNETES = "kubernetes";
+	private static final String TARGETTYPE_POLICYDEFINITIONS = "policydefinitions";
+	private static final String TARGETTYPE_DEFENDER = "defender";
+
+
+
+
```

Comment:
leave only single empty line.
general rule: if you want to separate - use single empty line, not more then one.